src/Features/CSharp/Portable/CodeFixes/ImplementAbstractClass/ImplementAbstractClassCodeFixProvider.cs

```diff
@@ -73,7 +73,7 @@ public sealed override FixAllProvider GetFixAllProvider()
         // internal for testing purposes.
         internal static string GetCodeActionId(string assemblyName, string abstractTypeFullyQualifiedName)
         {
-            return CSharpFeaturesResources.ImplementAbstractClass + ";" +
+            return CSharpFeaturesResources.Implement_Abstract_Class + ";" +
```

Comment:
oh the identifier just matches the text it refers to, including hte casing. So, its the text thats caps on all words, eh?

---
In reply to: [71400121](https://github.com/dotnet/roslyn/pull/12595#discussion_r71400121) [](ancestors = 71400121)